app-modules/alert/src/Observers/AlertObserver.php

```diff
@@ -74,9 +71,7 @@ public function deleted(Alert $alert): void
         $user = auth()->user();
 
         if ($user) {
-            Cache::tags([match ($alert->concern_type) {
-                app(Contact::class)->getMorphClass() => "user-{$user->getKey()}-contact-alerts",
-            }])->flush();
+            Cache::tags(["user-{$user->getKey()}-contact-alerts"])->flush();
```

Comment:
This could have to do with this branch being not up-to-date with the base branch `main`. So perhaps rebasing this on `main` and force pushing up will resolve this.

If after you do that, this still doesn't have `{}` around it, please add that.

```suggestion
            Cache::tags(["{user-{$user->getKey()}-contact-alerts}"])->flush();
```